contracts/cheddar_5x5_tic_tac_toe/src/game.rs

```diff
@@ -69,7 +73,7 @@ impl Game {
         let piece_2 = piece_1.other();
         (
             Player::new(piece_1, account_id_1),
-            Player::new(piece_2, account_id_2)
+            Player::new(piece_2, account_id_2),
```

Comment:
probably we can remove the `Player` structure and always assume that first player is `O` and second is `X` (alphabetic order). Will simplify the game and reduce code.